src/screens/SavedEventListScreen/NoSavedEvents.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import React, { PureComponent } from "react";
+import { Image, StyleSheet, View } from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import Text from "../../components/Text";
+import ContentPadding from "../../components/ContentPadding";
+import ButtonPrimary from "../../components/ButtonPrimary";
+import whiteCheck from "../../../assets/images/whiteCheck.png";
+import iconSave from "../../../assets/images/save.png";
+import noSavedEvents from "../../../assets/images/howToSaveEvents.png";
+import text from "../../constants/text";
+import { cardBgColor, titleTextColor } from "../../constants/colors";
```

Comment:
We have slowly been migrating away from using abstracted colors and toward using the named colors from Zeplin. Can you do this here too?